cmake/modules/OPAE.cmake

```diff
@@ -57,7 +57,7 @@ include(FindUdev)
 
 include(OPAECompiler)
 if(OPAE_BUILD_TESTS)
-    find_package(GTest 1.8.0)
+    find_package(GTest 1.11.0)
```

Comment:
Should we store this in a cmake variable? That way when we change versions we only have to change it in one place?

Reply:
Where should we store it?

Reply:
done